simexpal/base.py

```diff
@@ -693,64 +693,191 @@ def check_available(self):
 	def install(self):
 		from . import instances
 
-		global DID_WARN_KONECT
-
-		if self.check_available() or self.is_fileless:
+		if self.is_fileless:
```

Comment:
I moved ``check_available()`` to the later line number 702. In this way we can only postprocess instances, if they are already available (possible reasons: its a local instance or postprocessing was added after downloading the instances)